packages/react-charting/docs/ComponentTests/DonutChartTestPlan.md

```diff
@@ -0,0 +1,48 @@
+<div align="center">
+
+# Component testing - Donut chart test plan
```

Comment:
Call the containing directory TestPlans, create a folder called DonutChart and call this file ComponentTests.md